### PR TITLE
Add vanilla yosys with UHDM plugin

### PR DESCRIPTION
**Pull request by RRozak**

### CODE REVIEW

File: tools/runners/VanillaYosysPlugin.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+#
+# SPDX-License-Identifier: ISC
+
+import os
+import sys
+
+from BaseRunner import BaseRunner
+
+
+class VanillaYosysPlugin(BaseRunner):
```
Comment (tgorochowik) [inline tools/runners/VanillaYosysPlugin.py:18]: We need to make sure it is clear that it's the UHDM plugin (as there are many different yosys plugins in general).

File: .github/workflows/sv-tests-ci.yml [hunk 2]
```diff
@@ -39,11 +39,16 @@ jobs:
             deps: autoconf autotools-dev bison default-jre flex libfl-dev cmake pkg-config tclsh uuid-dev
             runners_filter: UhdmVerilator
           - name: yosys
-            deps: bison clang tcl-dev flex libfl-dev
+            deps: bison clang tcl-dev flex libfl-dev pkg-config libreadline-dev
+            runners_filter: Yosys
```
Comment (kamilrakoczy) [inline .github/workflows/sv-tests-ci.yml:43]: With changed name of ``vanilla-yosys-uhdm-plugin`` binary, this is no longer needed.

**Review by kamilrakoczy — CHANGES_REQUESTED** [review]
Please squash commits and make PR in ``yosys-uhdm-plugin-integration`` repository with all required changes.

File: .github/workflows/sv-tests-ci.yml (hunk 2 above)
Comment (RRozak) [reply .github/workflows/sv-tests-ci.yml]: Removed

**Review by kamilrakoczy — APPROVED** [review]

**Pull request merged**